kmp-library/build.gradle

```diff
@@ -1,6 +1,4 @@
-plugins {
-    id 'kotlin-multiplatform' version '1.3.11'
-}
+apply plugin: 'kotlin-multiplatform'
```

Comment:
I had to change the way we apply the plugin due to some build issues after applying ktor.